src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -173,20 +173,21 @@ public override string ToString()
             ThrowHelper.ThrowForUnsupportedVectorBaseType<T>();
 
             int lastElement = Count - 1;
-            StringBuilder sb = StringBuilderCache.Acquire();
+            Span<char> initialBuffer = stackalloc char[64];
+            var sb = new ValueStringBuilder(initialBuffer);
             CultureInfo invariant = CultureInfo.InvariantCulture;
 
             sb.Append('<');
             for (int i = 0; i < lastElement; i++)
             {
-                sb.Append(((IFormattable)this.GetElement(i)).ToString("G", invariant))
-                 .Append(',')
-                 .Append(' ');
+                sb.Append(((IFormattable)this.GetElement(i)).ToString("G", invariant));
+                sb.Append(',');
+                sb.Append(' ');
```

Comment:
Same question/comment as earlier... I wonder if `sb.Append(", ");` would be better, just for simplicity reasons.